tempus_fugit_minecraft/model.py

```diff
@@ -6,6 +6,7 @@
 from pyglet.gl import GL_QUADS
 from pyglet.graphics import TextureGroup, Batch
 from pyglet import image
+from tempus_fugit_minecraft.block import *
```

Comment:
I am never a fan of using wildcard imports, I would recommend to explicitly import what you are using. This way it makes it a lot easier to see where certain variables or methods are coming from.